src/components/CompletedTransactionsList/CompletedTransactionsList.story.css

```diff
@@ -0,0 +1,9 @@
+#root .CompletedTransactionsList .no-results {
```

Comment:
Why is `#root` being used? - Can we just stick with the component selector to restrict the scope, or is this not sufficient?

Reply:
That styling was added before I touched it. I agree that there shouldn't be any need for it, especially if the styling is restricted to the storybook CSS.

Will remove.

Reply:
Done